### PR TITLE
Revise some error messages

### PR DESCRIPTION
**Pull request by fingolfin**
- Don't use trailing period
- Don't print values; the user can print them if they want to via the break loop
- Be more consistent in how errors are worded (more to be done)

### CODE REVIEW

**Review by maxhauck — APPROVED** [review]
Duly noted!

File: gap/ExtraspecialNormalizerMatrixGroups.gi [hunk 1]
```diff
@@ -661,11 +658,10 @@ BindGlobal("ExtraspecialNormalizerInSU",
 function(r, m, q)
     local F, result;
     if not r ^ m > 2 then
-        ErrorNoReturn("<r> ^ <m> must be at least 2 in the unitary case, but",
-                      " <r> = ", r, " and <m> = ", m);
+        ErrorNoReturn("<r> ^ <m> must be at least 2 in the unitary case,");
     elif not (q + 1) mod r = 0 or (IsEvenInt(r) and not (q + 1) mod 4 = 0) then
         ErrorNoReturn("<q> + 1 must be divisible by r (or 4 if r = 2) in the ",
-                      "unitary case, but <q> =", q, " and <r> =", r);
+                      "unitary case,");
```
Comment (TristanPfersdorff) [inline gap/ExtraspecialNormalizerMatrixGroups.gi:664]:
```suggestion
                      "unitary case");
```

File: gap/ExtraspecialNormalizerMatrixGroups.gi [hunk 2]
```diff
@@ -661,11 +658,10 @@ BindGlobal("ExtraspecialNormalizerInSU",
 function(r, m, q)
     local F, result;
     if not r ^ m > 2 then
-        ErrorNoReturn("<r> ^ <m> must be at least 2 in the unitary case, but",
-                      " <r> = ", r, " and <m> = ", m);
+        ErrorNoReturn("<r> ^ <m> must be at least 2 in the unitary case,");
```
Comment (TristanPfersdorff) [inline gap/ExtraspecialNormalizerMatrixGroups.gi:661]:
```suggestion
        ErrorNoReturn("<r> ^ <m> must be at least 2 in the unitary case");
```

**Review by TristanPfersdorff — CHANGES_REQUESTED** [review]
Looks good, I'll be sticking to these conventions in the future. Just found a tiny imperfection :)

**Review by TristanPfersdorff — APPROVED** [review]

**Pull request merged**